backends/imgui_impl_vulkan.cpp

```diff
@@ -1858,6 +1863,11 @@ static void ImGui_ImplVulkan_SwapBuffers(ImGuiViewport* viewport, void*)
     ImGui_ImplVulkanH_Window* wd = &vd->Window;
     ImGui_ImplVulkan_InitInfo* v = &bd->VulkanInitInfo;
 
+	if (!wd->CanPresent) {
+		wd->CanPresent = true;
+		return;
+	}
+
     VkResult err;
```

Comment:
Can you also clarify how this is needed?
Our main.cpp example do a similar thing via setting `g_SwapChainRebuild = true` but it is primarily because the swap chain recreation is deferred to later.
Can you confirm what happens if you attempt the `vkQueuePresentKHR()` ? for my records. Thanks!

Reply:
When I get some time to try this, I'll let you know how it turned out